create-serverless-service/index.js

```diff
@@ -90,7 +90,6 @@ module.exports.handler = (event, context, cb) => {
             username: user_id,
             admin_group: userlist,
             domain: event.body.domain,
-            auth_token: event.headers.Authorization,
```

Comment:
Since we are removing this here, 
- can we remove the same during request validation?
- update unit tests?

Reply:
Removed the request validation for headers also updated the test case